generator/config/stage/densify.yaml

```diff
@@ -28,3 +28,29 @@ arguments:
             - object # Range
         description: |
             Specification for range based densification.
+tests:
+    -
+        name: 'Densify Time Series Data'
+        link: 'https://www.mongodb.com/docs/manual/reference/operator/aggregation/densify/#densify-time-series-data'
+        pipeline:
+            -
+                $densify:
+                    field: 'timestamp'
+                    range:
+                        step: 1
+                        unit: 'hour'
```

Comment:
We need to create an enum for time unit, used in a lot of places.

Reply:
Is this ticketed?

Reply:
Here [PHPLIB-1370](https://jira.mongodb.org/browse/PHPLIB-1370)